_build/maintainer_guide.rst

```diff
@@ -0,0 +1,332 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and is managed by the `Symfony Docs team`_. This article explains in detail some
+of those management tasks, so it's only useful for maintainers and not regular
+readers or Symfony developers.
+
+Reviewing Pull Requests
+-----------------------
+
+All the recommendations of the `Symfony's respectful review comments`_ apply,
+but there are extra things to keep in mind for maintainers:
+
+* Always be nice in all interactions with all contributors.
+* Be extra-patient with new contributors (they show a special badge on GitHub).
```

Comment:
Are the two items above really exclusive for the documentation repository?

Reply:
They are not exclusive ... but we shouldn't care ... this article is only for doc mergers, not code mergers.